pkg/ctl/upgrade/nodegroup.go

```diff
@@ -33,6 +33,7 @@ func upgradeNodeGroupCmd(cmd *cmdutils.Cmd) {
 		fs.StringVar(&options.KubernetesVersion, "kubernetes-version", "", "Kubernetes version")
 		fs.StringVar(&options.LaunchTemplateVersion, "launch-template-version", "", "Launch template version")
 		fs.BoolVar(&options.ForceUpgrade, "force-upgrade", false, "Force the update if the existing node group's pods are unable to be drained due to a pod disruption budget issue")
+		fs.StringVar(&options.ReleaseVersion, "release-version", "", "AMI version of the EKS optimized AMI to use")
```

Comment:
this option is only for managed ngs, correct?

Reply:
The command itself is only for managed nodegroups.

Reply:
orly? lol i totally did not get that from the cli help, how do people even use eksctl? 😂